### ISSUE TITLE
Color contrast issue with terra-utility example

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-framework#506]
# Issue

## Description
Color contrast in [this example](https://github.com/cerner/terra-framework/blob/master/packages/terra-application-utility/src/terra-dev-site/doc/example/ApplicationHeaderUtilityExample.jsx) doesn't meet the recommended contrast ratio.

## Steps to Reproduce

1. Navigate to https://engineering.cerner.com/terra-ui/#/components/terra-application-utility/application-utility/application-utility
2. Use color contrast tools to inspect the color ratio.

## Expected Behavior
Examples are accessible

## Possible Solution
Work with UX (@neilpfeiffer) to get [color for the background in the example](https://github.com/cerner/terra-framework/blob/master/packages/terra-application-utility/src/terra-dev-site/doc/example/ApplicationHeaderUtilityExample.jsx#L29) that meets a recommended contrast ratio with white text.
